ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -489,6 +493,24 @@
         }
     });
 
+    function registerRestEndpointUrlValidator() {
+        $(window).adaptTo("foundation-registry").register("foundation.validation.validator", {
+            selector: "[data-rest-endpoint-url-validation]",
+            validate: (el) => {
+                const url = el.value;
+
+                // Regex to validate absolute URLs starting with http:// or https:// only
+                const absoluteUrlPattern = /^(https?:\/\/)([a-z0-9\-]+\.)*[a-z0-9\-]+(\.[a-z]{2,})(:[0-9]{1,5})?(\/.*)?$/i;
+
+                if (!absoluteUrlPattern.test(url)) {
+                    return Granite.I18n.getMessage(
+                        "Enter a valid URL for the POST request. Ensure the URL starts with 'https://' or 'http://' and follows this structure: https://www.server.com/path/to/resource"
```

Comment:
Is this reviewed by doc team?
For translation - @rismehta we need this in i18n repo ?

Reply:
Yes, Khushwant has provide the string and it has been reviewed by him.